### PR TITLE
JBDS-4423 Fuse Tools installer integrated

### PR DESCRIPTION
**Pull request by dgolovin**

### CODE REVIEW

File: browser/model/devstudio.js [hunk 1]
```diff
@@ -9,14 +9,15 @@ import Logger from '../services/logger';
 import JdkInstall from './jdk-install';
 
 class DevstudioInstall extends InstallableItem {
-  constructor(installerDataSvc, targetFolderName, downloadUrl, fileName, devstudioSha256, additionalLocations, additionalIus) {
-    super(DevstudioInstall.KEY, downloadUrl, fileName, targetFolderName, installerDataSvc, true);
+  constructor(keyName, installerDataSvc, targetFolderName, downloadUrl, fileName, sha256sum, additionalLocations, additionalIus, useDownload) {
```
Comment (jrichter1) [inline browser/model/devstudio.js]: Any particular reason to use keyName in constructor here, while other model classes use a combination of static Class.KEY and this.keyName?

Comment (dgolovin) [inline browser/model/devstudio.js]: the reason is reusing it for fuse tools installation with additionalIus (IU - Installation Unit) configured in addition

**Comment by jrichter1** [comment]
I've run into a couple of problems while testing this:
 - fuse can be selected without devstudio (particularly deadly in online installer or if you did not bother with installing java)
 - fuse ignores bundled devstudio jar and always looks for it in the download folder, which is disastrous in the offline installer for a change

**Comment by jrichter1** [comment]
Or rather it might be caused by the fact that devstudio filename gets updated in requirements.json during the build and fuse doesn't.

**Comment by dgolovin** [comment]
Thanks Jan, I am looking into both issues. First is related to the fact that we have no generic dependency resolution. I've added 'requires' attribute to every components where applicable, but fact that CDK should work with ether virtualbox or hyper-v holding. Hope I'll find solution and resolve this issue till my PTO starts.

Second seems to be simple bug to fix, because you already found the reason for it.

**Comment by dgolovin** [comment]
test this

**Pull request merged**